<ui version="4.0" >
 <class>ECSConversionDialog</class>
 <widget class="QDialog" name="ECSConversionDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Mesh convertion to MED</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="CaseLabel" >
       <property name="text" >
        <string>Case selection</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CaseCB" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="OptionsGroup" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QCheckBox" name="VolOptButton" >
        <property name="text" >
         <string>--volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="BordOptButton" >
        <property name="text" >
         <string>--boundary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Color2GroupButton" >
        <property name="text" >
         <string>--color-to-group</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="ResultLabel" >
       <property name="text" >
        <string>Result med file name</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ResultNameLE" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer name="spacer2_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>61</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ConvertBtn" >
       <property name="text" >
        <string>Convert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBtn" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>61</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>CancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>ECSConversionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ConvertBtn</sender>
   <signal>clicked()</signal>
   <receiver>ECSConversionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
